integration_test/stripe_full.sh

```
@@ -0,0 +1,165 @@
+    if [ ! -s "$f" ]; then
+      fail "Generated file $f is empty (codegen regression)."
+    fi
+  done <<< "$files"
+
+  while IFS= read -r f; do
+    if ! head -n1 "$f" | grep -q "// Code generated by oaspec"; then
+      fail "Generated file $f is missing the codegen provenance header."
+    fi
+  done <<< "$files"
+
+  # shellcheck disable=SC2086 -- intentional word-splitting on find output
```

Shellcheck (0.11.0)

[error] Couldn't parse this shellcheck directive. Fix to allow more checks.

(SC1073)